src/cf-forms/src/organisms/form.less

```diff
@@ -1,4 +1,8 @@
 .o-form {
+    .m-form-field + .m-form-field {
+        margin-top: 10px;
```

Comment:
Also below, we have `.m-field + .m-field { margin-top: 10px; }` when it's within a fieldset. Are we supposed to have both `.m-form-field` and `.m-field`?